proxy/src/main/java/ru/leymooo/botfilter/captcha/generator/CaptchaPainter.java

```diff
@@ -14,14 +14,16 @@
 import java.awt.image.ConvolveOp;
 import java.awt.image.Kernel;
 import java.util.Random;
+import lombok.RequiredArgsConstructor;
 
+@RequiredArgsConstructor
 public class CaptchaPainter
 {
 
     private final int width = 128;
     private final int height = 128;
     private final Color background = Color.WHITE;
-    private final Random rnd = new Random();
+    private final Random rnd;
```

Comment:
This will just throw NPE's, please fix it!

Reply:
> This will just throw NPE's, please fix it!

The class is annotated lombok with @RequiredArgsConstructor annotation
It generates a constructor where there is Random

New CaptchaPainter instances are created in the CaptchaGenerationTask class always with Random instances inside